cmd/minikube/cmd/start_flags.go

```diff
@@ -93,9 +93,9 @@ const (
 	waitTimeout             = "wait-timeout"
 	nativeSSH               = "native-ssh"
 	minUsableMem            = 1024 // Kubernetes will not start with less than 1GB
-	minRecommendedMem       = 2000 // Warn at no lower than existing configurations
+	minRecommendedMem       = 2048 // Warn at no lower than existing configurations
 	minimumCPUS             = 2
-	minimumDiskSize         = 2000
+	minimumDiskSize         = 2048
```

Comment:
This is not about mebibytes

Reply:
> > This patch simply calculates disk size in MiB and assigns the correct unit in the domain configruation for libvirt.
> 
> This PR does a whole lot more than just change the output from MB to MiB, though.
> 
> Including changing the memory allocation from 2000 and disk allocation from 20000
> 
> It would be easier to review, if you opened a issue first describing what you want to "fix"
> 
> I think the libvirt unit mismatch was something we inherited from upstream (kvm) actually:
> 
> [dhiltgen/docker-machine-kvm#54](https://github.com/dhiltgen/docker-machine-kvm/issues/54)

That is true, and my earlier change didn't include any changes to start_flags... I could easily drop those.. they are unnecessary. That was mainly to align allocations to GiB... that should probably be in another PR after opening up an issue.